### DISCUSSION TITLE
Spreading the word about the moloch theme

### DISCUSSION THREAD

**Discussion by samcarter** [jolars/moloch#13] [General]
The German TeX user meeting is going to take place next week. I originally planned to mention your theme in my talk about beamer, but as there was still some room in the timetable, I decided to do a quick lightning talk about it, in which I will recount, from the perspective of a user,  how to switch from metropolis to moloch.

In case you'd like to take a look, the slides are at:

https://github.com/samcarter/samcarter/raw/main/ressouces/Dante2024_Moloch.pdf

(sorry, the slides are in German, I can provide translations if you are interested in any slide in particular. Also if you would like to have the source code of the slides, please let me know and I'll sent you a link to the repository)

---

I was wondering if doing something similar for the international TeX user group meeting (Prague, July 19-21 https://tug.org/tug2024/ ) might be a good way to spread the word about your theme. Do you happen to be there? If not, I could do a similar talk from the perspective of a user.

**Comment by jolars** [comment]
Nice, thanks for spreading the word! Nice Metropolis + logo slide too! But please make sure to not give me too much credit for the theme. My part is minor compared to what Matthias did.

No, I unfortunately won't be there. But I would of course be very happy for you to keep spreading the word :)

Your slides make me realize that removing the high-contrast version of the theme probably was a mistake. I think I will bring it back again.

**Comment by samcarter** [comment]
> Nice, thanks for spreading the word! Nice Metropolis + logo slide too! But please make sure to not give me too much credit for the theme. My part is minor compared to what Matthias did.

I  think it is fair to share the credit between the two of you. Yes, Matthias Vogelgesang wrote a very powerful theme which is used by many people, but your fork keeps it functional with the current beamer+kernel. 

> No, I unfortunately won't be there. But I would of course be very happy for you to keep spreading the word :)

Great!  I'll submit a short talk then and keep you in the loop!
 
> Your slides make me realize that removing the high-contrast version of the theme probably was a mistake. I think I will bring it back again.

\begin{joke}
   Please don't fix too many of the differences between now and next week or the talk will be over very quickly :)
\end{joke}
